packages/opentelemetry-exporter-collector/src/platform/browser/util.ts

```diff
@@ -28,7 +28,7 @@ export function sendWithBeacon(
   onSuccess: () => void,
   onError: (error: collectorTypes.CollectorExporterError) => void
 ) {
-  if (navigator.sendBeacon(url, body)) {
+  if (navigator.sendBeacon(url, new Blob([body], { type: "application/json" }))) {
```

Comment:
This function is exported as a public function from the package, however its possible that it don't want to them data as a json. So we should allow user to give a variable that represent the contentType (and fallback to json by default)

Reply:
@vmarchaud I added `contentTypeBeacon` in CollectorExporterConfigBase. So,  we can configure content-type for blob and beacon

Reply:
there are already utilities inn the code base like parseHeaders that already help for type checkimg and values of types. 
also, i think adding behavior to sendBeacon may be hard to maintain. Given i added behavior to sendWithXhr in think it was a viable solution some days ago. Therefore i created a PR that maybe solves here: https://github.com/open-telemetry/opentelemetry-js/pull/2380 
(Maybe, locally my tests all pass , but who knows in the CI :-) )

// cc. @jufab , @vmarchaud

Reply:
@niko-achilles thx for your inspiration and for the spread operator (I don't know this one 😊)
Finally, i prefer to use a BlobPropertyBag than header (this is not header in definition of Blob...)

Reply:
@jufab  Bien fait . nice (personal opinion) , that sendWithBeacon has no logic in context of correcting configuration . 
Respecting code-owners when approve this commit , i should  propose a refactoring on headers when xhr is used.

Reply:
Even though it's not a header, the internals of `withBeacon` will only use what it needs to i would prefer going ahead with #2380 if @jufab don't mind

Reply:
@vmarchaud Sorry but I'm not understand (I'm not fluent in English but I try to improve myself). 
Should I close this PR and you want to fix issue #2321 with the PR #2380 ? 
It's not a problem for me if I understood correctly.

Reply:
@jufab Oui a la base c'etait ce que je voulais dire mais apparament l'auteur de #2380 (qui au debut devait regler ce probleme) va proposer autre chose donc on peut continuer avec cette PR :)

(EN: Yeah that was i meant, but the author of #2380 (which would have solved this issue) might refactor  his PR and let this one fixes this specific issue, so we can continue with this one)

Reply:
@vmarchaud Ok! (And thx for the FR/EN version 😊 )